src/sites/twitch-twilight/modules/player.jsx

```diff
@@ -860,11 +865,10 @@ export default class Player extends Module {
 
 	pipPlayer(inst, e) {
 		const video = inst.props.mediaPlayerInstance?.mediaSinkManager?.video;
-		if ( ! video || ! document.pictureInPictureEnabled )
+		if ( ! video || ( ! document.pictureInPictureEnabled && ! window.safari ) )
 			return;
 
-		if ( e )
-			e.preventDefault();
+		e?.preventDefault?.();
```

Comment:
This expands into more verbose code than what you are replacing. e should always be an Event if set and should always have a preventDefault.